genic/save.c

```diff
@@ -125,13 +125,13 @@ void saveheader(BigFile * bf) {
             (big_block_set_attr(&bheader, "Time", &InitTime, "f8", 1)) ||
             (big_block_set_attr(&bheader, "Redshift", &redshift, "f8", 1)) ||
             (big_block_set_attr(&bheader, "BoxSize", &Box, "f8", 1)) ||
-            (big_block_set_attr(&bheader, "OmegaM", &Omega, "f8", 1)) ||
-            (big_block_set_attr(&bheader, "OmegaB", &OmegaBaryon, "f8", 1)) ||
-            (big_block_set_attr(&bheader, "OmegaL", &OmegaLambda, "f8", 1)) ||
+            (big_block_set_attr(&bheader, "OmegaM", &CP.Omega0, "f8", 1)) ||
```

Comment:
I recently started to like the CLASS set of names, LAMBDA, CDM, B, UR, NCDM, and NCDM_TOT. The word neutrino doesn't even appear.